src/Map/src/Bridge/Leaflet/src/LeafletOptions.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Leaflet;
+
+use Symfony\UX\Map\Bridge\Leaflet\Option as LeafletOption;
+use Symfony\UX\Map\MapOptionsInterface;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class LeafletOptions implements MapOptionsInterface
+{
+    public function __construct(
+        private ?LeafletOption\TileLayer $tileLayer = new LeafletOption\TileLayer(
```

Comment:
Should not be nullable

Reply:
Why?

Reply:
Then your toArray method is dangerous (if tileLayer is null)

Reply:
Oh, yeah you're right, 1:40 AM it's starting to be late :p 

I had the same reflection some days ago for GoogleOptions and I removed `?`, but forgout about LeafletOptions